### PR TITLE
Add ITKPhaseCorrelationImageRegistration

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

**Review by thewtex — APPROVED** [review]

File: MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp [hunk 1]
```diff
@@ -0,0 +1,244 @@
+// File automatically generated
```
Comment (dzenanz) [inline MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp:1]: Should this be removed now? Or is this file occasionally automatically regenerated?

File: MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.h [hunk 2]
```diff
@@ -0,0 +1,132 @@
+// File automatically generated
```
Comment (dzenanz) [inline MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.h:1]: remove comment?

File: MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp (hunk 1 above)
Comment (fbudin69500) [reply MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp:1]: The file is regenerated automatically. This is helpful if the script that generates the files is updated for example, like in this PR.

File: MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.h (hunk 2 above)
Comment (fbudin69500) [reply MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.h:1]: Same as other comment.

File: MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp (hunk 1 above)
Comment (fbudin69500) [reply MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp:1]: Yes.

Comment (fbudin69500) [reply MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp:1]: See #3

**Pull request merged**